src/Core/Models/Api/Request/Organizations/OrganizationSsoRequestModel.cs

```diff
@@ -40,47 +39,11 @@ public class SsoConfigurationDataRequest : IValidatableObject
     {
         public SsoConfigurationDataRequest() {}
 
-        public SsoConfigurationDataRequest(SsoConfigurationData configurationData)
-        {
-            ConfigType = configurationData.ConfigType;
-            UseCryptoAgent = configurationData.UseCryptoAgent;
-            CryptoAgentUrl = configurationData.CryptoAgentUrl;
-            Authority = configurationData.Authority;
-            ClientId = configurationData.ClientId;
-            ClientSecret = configurationData.ClientSecret;
-            MetadataAddress = configurationData.MetadataAddress;
-            RedirectBehavior = configurationData.RedirectBehavior;
-            GetClaimsFromUserInfoEndpoint = configurationData.GetClaimsFromUserInfoEndpoint;
-            IdpEntityId = configurationData.IdpEntityId;
-            IdpBindingType = configurationData.IdpBindingType;
-            IdpSingleSignOnServiceUrl = configurationData.IdpSingleSignOnServiceUrl;
-            IdpSingleLogoutServiceUrl = configurationData.IdpSingleLogoutServiceUrl;
-            IdpArtifactResolutionServiceUrl = configurationData.IdpArtifactResolutionServiceUrl;
-            IdpX509PublicCert = configurationData.IdpX509PublicCert;
-            IdpOutboundSigningAlgorithm = configurationData.IdpOutboundSigningAlgorithm;
-            IdpAllowUnsolicitedAuthnResponse = configurationData.IdpAllowUnsolicitedAuthnResponse;
-            IdpDisableOutboundLogoutRequests = configurationData.IdpDisableOutboundLogoutRequests;
-            IdpWantAuthnRequestsSigned = configurationData.IdpWantAuthnRequestsSigned;
-            SpNameIdFormat = configurationData.SpNameIdFormat;
-            SpOutboundSigningAlgorithm = configurationData.SpOutboundSigningAlgorithm ?? SamlSigningAlgorithms.Sha256;
-            SpSigningBehavior = configurationData.SpSigningBehavior;
-            SpWantAssertionsSigned = configurationData.SpWantAssertionsSigned;
-            SpValidateCertificates = configurationData.SpValidateCertificates;
-            SpMinIncomingSigningAlgorithm = configurationData.SpMinIncomingSigningAlgorithm ?? SamlSigningAlgorithms.Sha256;
-            AdditionalScopes = configurationData.AdditionalScopes;
-            AdditionalUserIdClaimTypes = configurationData.AdditionalUserIdClaimTypes;
-            AdditionalEmailClaimTypes = configurationData.AdditionalEmailClaimTypes;
-            AdditionalNameClaimTypes = configurationData.AdditionalNameClaimTypes;
-            AcrValues = configurationData.AcrValues;
-            ExpectedReturnAcrValue = configurationData.ExpectedReturnAcrValue;
-        }
```

Comment:
I guess we just aren't using this anywhere? Any specific reasons it was removed on this PR?

Reply:
Not really. I touched upon the code, in this and the previous PR so it seemed a good opportunity to remove it.